docs/Glossary.md

```diff
@@ -1,3 +1,46 @@
 # Glossary
 
-See [Terminologies](/introduction/Terminologies.html).
+## App
```

Comment:
I wouldn't mind if you delete this page entirely. but please let's keep Terminologies page where it is already.